src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/toolbarActions/AbstractFormToolbarAction.js

```diff
@@ -3,7 +3,7 @@ import type {Node} from 'react';
 import {ResourceFormStore} from '../../../containers/Form';
 import type {ToolbarAction, ToolbarItemConfig} from '../../../containers/Toolbar/types';
 import Router from '../../../services/Router';
-import Form from '../../../views/Form';
+import Form from '../../../views/Form/Form';
```

Comment:
When looking at this I realized that `../Form` would be the easier import :see_no_evil: